gocov/test.go

```diff
@@ -86,12 +90,24 @@ func runTests(args []string) error {
 		// the JSON coverage output.
 		cmd.Stdout = os.Stderr
 		cmd.Stderr = os.Stderr
-		err := cmd.Run()
+
+		commandChan <- cmd
+	}
+
+	for i := 0; i < runtime.NumCPU(); i++ {
```

Comment:
... it just occurred to me that I may have led you astray. The testflag code defines which flags to pass to the test commands, but we really just want it in the gocov process. Sorry, my memory was hazy.

So I think the testflag.Split function needs to change a bit, and needs to grow the ability to parse flags for the gocov driver itself. To start with, only "-p" would be interpreted by gocov.